src/main.c

```diff
@@ -337,6 +341,11 @@ int main(void) {
         .filename = filename,
     };
 
+    k_thread_create(&radio_thread, radio_thread_stack,
+                    K_THREAD_STACK_SIZEOF(radio_thread_stack), radio_receiver,
+                    NULL, NULL, NULL, K_HIGHEST_APPLICATION_THREAD_PRIO, 0,
```

Comment:
I'm sorry, I just remembered, when used like this, this makes the thread [cooperative (non-preemptible)](https://docs.zephyrproject.org/latest/kernel/services/threads/index.html#thread-priorities). I'm not sure we want this, perhaps we should play it safe and choose the highest preemptible thread priority.